src/views/open-api-docs-view/components/property-details/index.tsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import classNames from 'classnames'
+import { MdxInlineCode } from 'components/dev-dot-content/mdx-components'
+import Badge from 'components/badge'
+import { IconCornerDownRight16 } from '@hashicorp/flight-icons/svg-react/corner-down-right-16'
+import s from './property-details.module.css'
+
+/**
+ * Types are shared by the "base" and "deeper" variations of
+ * the property details component.
+ */
+export type PropertyDetailsProps = {
```

Comment:
Any particular reason for using `type` instead of `interface` here?

Reply:
No particular reason, just slipped my mind 😅 Updated to `interface`! 👍